wip-0018.md

```diff
@@ -124,8 +124,9 @@ A reference implementation for the proposed protocol improvement can be found in
 
 ## Adoption Plan
 
-An adoption plan will be proposed upon moving this document from the _Draft_ stage to the _Proposed_ stage.
+An activation date is proposed for September 21 2021 at 9am UTC, that is, protocol epoch #656640.
 
+From the perspective of TAPI, this proposal will be signaled along with the WIP-0017. This proposal adheres to the base implicit signal, and its first signaling epoch is also set to that of WIP-0017.
```

Comment:
This won't adhere to the base signal but rather adhere to the signal of `WIP-0017`, let's explicitly state the bit mask here!

Reply:
True! I missed that :sweat_smile: